src/js/views/patients/worklist/worklist-list.scss

```diff
@@ -52,6 +52,10 @@
   white-space: normal;
 }
 
+.worklist-list__meta {
```

Comment:
This fixes a wonky ellipsis that displays in the meta cell.

<img width="441" alt="Screen Shot 2020-11-04 at 10 45 27 AM" src="https://user-images.githubusercontent.com/1137390/98173855-a7979400-1ea8-11eb-84c1-1c440d7a712b.png">